unused/war_over_the_heavenly_horses.xml

```diff
@@ -0,0 +1,8 @@
+Only Emperor Wu decided to finally put an end to it and declare war on the Xiognu. And to counter the Xiognu horsemen, he planned to raise a big cavalry force himself. There was only one problem: one of the few natural resources that the mighty Han dynasty lacked were powerful horses suitable for cavalry warfare. Local breeds were simply too small and weak. And large parts of the Empire were everything but ideal for horses grazing.
+But the news Zhang Qian brought came in very convenient: Zhang Qian was a Han envoy sent by Emperor Wu himself to explore and find allies in the West. He had, after over a decade, finally returned and although failed to make allies, brought invaluable information about previously unknown lands. He had, among others, also come to Greco-Bactria. Greco-Bactria was, because Alexander the Great had conquered this region and established colonies there not even 200 years prior, inhabited by predecessors of Alexander's soldiers, hence by Greeks. And Zhang Qian reported seeing extremely strong horses there—exactly what the Han needed. The horses he had seen were the mystical breed of so-called Ferghana horses. They were comparably small and had short legs, but they were very fast, powerful, and endurable. They were even said to have sweated blood. In fact, Zhang Qian was so thrilled by the Ferghana horses that he called them “Heavenly Horses”. Emperor Wu immediately sent an envoy, trying to buy some horses. However, the Greeks declined and killed the envoy. It is not entirely sure why, but the Ferghana horses were probably considered national pride and not to be sold. Emperor Wu sent his general Li Guangli with army of 20.000 infantry and 6.000 cavalry to take the horses by force. However, to get to Greco-Bactria, Li Guangli had to traverse the hostile Gobi and Taklamakan deserts. He also got into fights with the local Tocharians. And although he eventually arrived at its target, his army was very decimated and Li Guangli was forced to withdraw back to China. 
+Emperor Wu subsequently sent Li Guangli with an even larger army to attack Greco-Bactria. This time consisting of 60.000 infantry and 30.000 cavalry and taking over 100.000 cattle to avoid the food shortages faced during the first attempt. However, Li Guangli's army still suffered high casualties during the trip, and upon arriving had lost every second soldier. Despite that, the Han managed to defeat the Greeks and seize 3.000 of the desired horses. The journey back to China was no easy one as well. Li Guangli lost a large portion of the Heavenly Horses, and only every tenth soldier made it through the entire expedition.
```

Comment:
```suggestion
Emperor Wu subsequently sent the general back with an even larger army, this time consisting of 60,000 infantry and 30,000 cavalry and taking over 100,000 cattle to avoid the food shortages faced during the first attempt. However, Li Guangli's army still suffered high casualties during the trip, and upon arriving had lost every second soldier. Despite that, the Han managed to defeat the Greeks and seize 3,000 of the desired horses. The journey back to China was not no one as well. Li Guangli lost a large portion of the Heavenly Horses, and only every tenth soldier made it through the entire expedition.
```